2012-02-16  David S. Miller  <davem@davemloft.net>
 
	[BZ #11494]
	* sysdeps/unix/sysv/linux/sparc/bits/fcntl.h (O_FSYNC): Define.

---
 sysdeps/unix/sysv/linux/sparc/bits/fcntl.h |    1 +
 1 file changed, 1 insertion(+)

--- a/sysdeps/unix/sysv/linux/sparc/bits/fcntl.h
+++ b/sysdeps/unix/sysv/linux/sparc/bits/fcntl.h
@@ -40,6 +40,7 @@
 #define O_TRUNC		0x0400	/* not fcntl */
 #define O_EXCL		0x0800	/* not fcntl */
 #define O_SYNC		0x802000
+#define O_FSYNC		O_SYNC
 #define O_NONBLOCK	0x4000
 #define O_NDELAY	(0x0004 | O_NONBLOCK)
 #define O_NOCTTY	0x8000	/* not fcntl */
